hack/test_e2e.go

```diff
@@ -0,0 +1,944 @@
+
+	k8s_context := "kind-" + cluster_name
+	kubectl 	:= bin_dir + "/kubectl"
+	kind		:= bin_dir + "/kind"
+	
+	artifacts_directory := os.Getenv("E2E_ARTIFACTS")
+	E2E_BACKEND := os.Getenv("E2E_BACKEND")
+	E2E_DEPLOYMENT_MODEL := os.Getenv("E2E_DEPLOYMENT_MODEL")
+
+	_, err := os.Stat(bin_dir)
+	if err !=nil && os.IsNotExist(err) {
+		log.Fatal(err)
+	}
+
+	_, err = os.Stat(kubectl)
+	if err != nil && os.IsNotExist(err) {
+		log.Fatal(err)
+	}
+
+	// Remove existing kube-proxy
+	cmd := exec.Command(kubectl, "--context", k8s_context, "delete", "--namespace", NAMESPACE, "daemonset.apps/kube-proxy")
+	err = cmd.Run()
+	if err != nil {
+		log.Fatalf("Cannot delete delete daemonset.apps kube-proxy\n", err)
+	}
+	fmt.Println("Removed daemonset.apps/kube-proxy.")
+
+	// Load kpng container image
+	// Preload kpng image 
+	cmd = exec.Command(kind, "load", "docker-image", KPNG_IMAGE_TAG_NAME, "--name", cluster_name)
+	err = cmd.Run()
+	if err != nil {
+		log.Fatalf("Error loading image to kind.\n", err)
+	}
+	fmt.Println("Loaded " + KPNG_IMAGE_TAG_NAME + " container image.")
+
+	// Create service account, clusterbinding and configmap for kpng
+	cmd = exec.Command(kubectl, "--context", k8s_context, "create", "serviceaccount", "--namespace", NAMESPACE, SERVICE_ACCOUNT_NAME)
+	err = cmd.Run()
+	if err != nil {
+		log.Fatalf("Error creating serviceaccount %s.\n", SERVICE_ACCOUNT_NAME, err)
+	}
+	fmt.Println("Created service account", SERVICE_ACCOUNT_NAME)
+
+	cmd = exec.Command(kubectl, "--context", k8s_context, "create", "clusterrolebinding", CLUSTER_ROLE_BINDING_NAME, "--clusterrole", CLUSTER_ROLE_NAME, "--serviceaccount", NAMESPACE + ":"+ SERVICE_ACCOUNT_NAME)
+	err = cmd.Run()
+	if err != nil {
+		log.Fatalf("Error creating clusterrolebinding %s.\n", CLUSTER_ROLE_BINDING_NAME, err)
+	}
+	fmt.Println("Created clusterrolebinding", CLUSTER_ROLE_BINDING_NAME)
+
+	cmd = exec.Command(kubectl, "--context", k8s_context, "create", "configmap", CONFIG_MAP_NAME, "--namespace", NAMESPACE, "--from-file", artifacts_directory + "/kubeconfig.conf")
+	err = cmd.Run()
+	if err != nil {
+		log.Fatalf("Error creating configmap", CONFIG_MAP_NAME)
+	}
+	fmt.Println("Created configmap", CONFIG_MAP_NAME)
+
+	// Deploy kpng from the template generated
+	E2E_SERVER_ARGS := "'kube', '--kubeconfig=/var/lib/kpng/kubeconfig.conf', 'to-api', '--listen=unix:///k8s/proxy.sock'"
+    E2E_BACKEND_ARGS := "'local', '--api=" + KPNG_SERVER_ADDRESS + "', 'to-" + E2E_BACKEND + "', '--v=" + KPNG_DEBUG_LEVEL + "'"
+
+	if E2E_DEPLOYMENT_MODEL == "single-process-per-node" {
+		E2E_BACKEND_ARGS="'kube', '--kubeconfig=/var/lib/kpng/kubeconfig.conf', 'to-local', 'to-" + E2E_BACKEND + "', '--v=" + KPNG_DEBUG_LEVEL + "'"
+	} 
+
+	E2E_SERVER_ARGS = "[" + E2E_SERVER_ARGS + "]"
+	E2E_BACKEND_ARGS = "[" + E2E_BACKEND_ARGS + "]"
+
+	// Setting vars for generate the kpng deployment based on template
+	_ = os.Setenv("kpng_image", KPNG_IMAGE_TAG_NAME) 
+    _ = os.Setenv("image_pull_policy", "IfNotPresent") 
```

Comment:
identation

Reply:
@aroradaman I don't know why this is so. On my repo there is no identation issues  https://github.com/emabota/kpng/blob/8f0f3eb5229fee8eca3ff0650c5946d91d75582b/hack/test_e2e.go#L708
Any idea what's causing this?

Reply:
use `go fmt hack/test_e2e.go`